mmv1/products/firebase/api.yaml

```diff
@@ -204,25 +204,41 @@ objects:
       - !ruby/object:Api::Type::String
         name: name
         description: |
-          The fully qualified resource name of the App, for example:
+          The fully qualified resource name of the AndroidApp, for example:
           projects/projectId/androidApps/appId
         output: true
       - !ruby/object:Api::Type::String
         name: displayName
         required: true
         description: |
-          The user-assigned display name of the App.
+          The user-assigned display name of the AndroidApp.
       - !ruby/object:Api::Type::String
         name: appId
         output: true
         description: |
-          The globally unique, Firebase-assigned identifier of the App.
+          The globally unique, Firebase-assigned identifier of the AndroidApp.
           This identifier should be treated as an opaque token, as the data format is not specified.
       - !ruby/object:Api::Type::String
         name: packageName
         description: |
           Immutable. The canonical package name of the Android app as would appear in the Google Play
           Developer Console.
+      - !ruby/object:Api::Type::Array
+        name: sha1Hashes
+        description: |
+          The SHA1 certificate hashes for the AndroidApp.
+        item_type: Api::Type::String
+      - !ruby/object:Api::Type::Array
+        name: sha256Hashes
+        description: |
+          The SHA256 certificate hashes for the AndroidApp.
+        item_type: Api::Type::String  
+      - !ruby/object:Api::Type::String
```

Comment:
This should be of type fingerprint.... making this field output only doesn't make sense.. If we are to include this field we need to send this to the api.

Reply:
referring to etag

Reply:
Done.